zerver/decorator.py

```diff
@@ -547,10 +547,6 @@ def web_public_view(
           web_public_view path has only been enabled for development purposes
           currently.
```

Comment:
I guess we should probably also drop this NOTE, if we're dropping the check below.